packages/remix-react/data.ts

```diff
@@ -5,10 +5,8 @@ import {
 
 /**
  * Data for a route that was returned from a `loader()`.
- *
- * Note: This moves to unknown in ReactRouter and eventually likely in Remix
  */
-export type AppData = any;
+export type AppData = unknown;
```

Comment:
Is it even useful to have this type exported?

We can't override it, so... 🤷‍♂️

Reply:
I'd be fine with that - we can run it by @mjackson and do that in a follow up PR if we want to drop all these aliases for `any`/`unknown` from the public API

Reply:
As long as it's merged _before_ v2 releases as a stable version, I'm fine with that

As removing them would be a breaking change as well